### ISSUE TITLE
Implement `exec.env-variables`

### ISSUE THREAD

**Issue by nikitabobko** [nikitabobko/AeroSpace#156]
Concept:
```
exec.inherit-env-variables = true
[exec.env-variables]
PATH = '/opt/homebrew/bin:/opt/homebrew/sbin:${PATH}'
```

**Comment by nikitabobko** [comment]
Alternative:
```
exec.inherit-env-variables = true
[exec.env-variables]
PATH = '/opt/homebrew/bin:/opt/homebrew/sbin:$PATH$'
```

Future possibilities:
```
[exec.env-variables]
PATH = { value = '/opt/homebrew/bin:/opt/homebrew/sbin:#PATH#', interpolation-char = '#' }
```

**Comment by nikitabobko** [comment]
Fixed in 0.9.0
